CHANGES.md

```diff
@@ -31,6 +31,8 @@ next
   `findlib.dynload`, automatically record linked in libraries and
   findlib predicates (#1172, @bobot)
 
+- Add support for promoting a selected list of files (#..., @diml)
```

Comment:
In case you forget, there's a missing PR number :)

Reply:
Indeed, fixed